pkg/minikube/constants/constants_windows.go

```diff
@@ -23,3 +23,12 @@ import (
 )
 
 var DefaultMountDir = homedir.HomeDir()
+
+// SupportedVMDrivers is a list of supported drivers on Windows.
+var SupportedVMDrivers = [...]string{
+	DriverVirtualbox,
+	DriverParallels,
```

Comment:
I am pretty sure that Parallels is not supported/can be used on Windows. Pretty sure it is only for Mac. Could you please confirm?

Reply:
Thanks!